src/components/App.js

```diff
@@ -28,6 +30,13 @@ const reducerCreate = params=> {
     };
 };
 
+const refreshOnBack = () => {
+  Actions.pop();
+  setTimeout(() => {
+    Actions.refresh();
+  }, 10);
```

Comment:
Confirmed.
//Delay 10ms to refresh previous scene when current scene is completely popped.